modules/mbedtls/crypto_mbedtls.cpp

```diff
@@ -181,6 +181,38 @@ Error X509CertificateMbedTLS::save(String p_path) {
 	return OK;
 }
 
+String X509CertificateMbedTLS::save_to_string() {
+	// Writes to a string buffer the value of the certificate.
+	String buffer;
+	mbedtls_x509_crt *crt = &cert;
```

Comment:
We probably need a way to check if the certificate is valid (has been loaded) and return an empty string instead of:
```
-----BEGIN CERTIFICATE-----
-----END CERTIFICATE-----
```

Reply:
I like it I'll see what I can find.

Reply:
So what I came up with is requiring at least one Base-64 character in the string length with a simple character count if check. I'd appreciate any feedback on this change. This doesn't verify any other returned values from the function are valid only that they are long enough to hold "some" amount of data.